libafl/src/executors/inprocess_fork/stateful.rs

```diff
@@ -119,7 +119,9 @@ where
                     self.inner.pre_run_target_child(fuzzer, state, mgr, input)?;
                     (self.harness_fn)(&mut self.exposed_executor_state, input);
                     self.inner.post_run_target_child(fuzzer, state, mgr, input);
-                    Ok(ExitKind::Ok)
+
+                    // post_run_target_child should make the process quit.
```

Comment:
I think the macro can take a parameter - in this case if this ever gets hit there's a chance it will get printed